### PR TITLE
Editing Chapter 2 (microbiome data)

### PR DESCRIPTION
**Pull request by jkc9886**
Made some edits:
1. grammatical changes 
2. punctuation marks.

### CODE REVIEW

File: inst/pages/04_containers.qmd [hunk 1]
```diff
@@ -34,9 +34,9 @@ way to bind several different data containers together in a single
 object. For example, we can bind microbiome data (in `TreeSE`
 container) with metabolomic profiling data (in `SE`) container, with
 (partially) shared sample metadata. This is convenient and robust for
-instance in subsetting and other data manipulation tasks. Microbiome
+instance, in subsetting and other data manipulation tasks. Microbiome
 data can be part of multiomics experiments and analysis strategies. We
-highlight how the methods used througout in this book relate to this
```
Comment (TuomasBorman) [inline inst/pages/04_containers.qmd:39]: typo

File: inst/pages/04_containers.qmd [hunk 2]
```diff
@@ -230,7 +230,7 @@ For information have a look at the [intro vignette](https://bioconductor.org/pac
  
    Option   Rows (features)    Cols (samples)               Recommended  
 ---------   --------------    ---------------  ------------------------
-   assays  	     match              match       Data transformations
```
Comment (TuomasBorman) [inline inst/pages/04_containers.qmd:233]: Do these extra spaces do something?

Comment (jkc9886) [reply inst/pages/04_containers.qmd:233]: They don't, I thought it would look more organized, but isn't so actually! I shall remove them.

Comment (antagomir) [reply inst/pages/04_containers.qmd:233]: This renders a table view, might help with editing to have it visually organized like that in this case.

Comment (jkc9886) [reply inst/pages/04_containers.qmd:233]: Sure, edited it the same way again. I believe giving two extra spaces in row 2 aligns everything in that row (that's what I did in commit 3 - I had removed it in commit 2 earlier).

Comment (jkc9886) [reply inst/pages/04_containers.qmd:233]: @TuomasBorman This is ready to be merged ( I have made the requested changes, could you please check once if anything's there?)

Comment (jkc9886) [reply inst/pages/04_containers.qmd:233]: Just a gentle reminder :)) @TuomasBorman

**Comment by jkc9886** [comment]
This PR is ready to be merged.

**Review by antagomir — APPROVED** [review]

**Comment by antagomir** [comment]
This seems OK - to merge, @TuomasBorman ?

**Review by TuomasBorman — APPROVED** [review]

**Pull request merged**